Vizzuality/heco-invest
# Update the static translation pipeline

This PR updates the static translation pipeline because the previous that was using Transifex Native wouldn't cover our needs for translated server-side rendered content.

In this new pipeline [FormatJS](https://formatjs.io/) is used to gather the source strings, and a git hook and GitHub action are used to push them and pull the translations from Transifex, which is now exclusively used as a database.

The whole pipeline is described in the README file below.

## Testing instructions

- Make sure the application runs locally in dev and production mode.
- Make sure the strings correctly appear in the application.
- Make sure the GitHub action triggered by this PR succeeded

What can only be tested when merged to `develop`:
- The GitHub action pushing the source strings

## Tracking

- [LET-146](https://vizzuality.atlassian.net/browse/LET-146).
- [LET-150](https://vizzuality.atlassian.net/browse/LET-150).
